src/handlers/ens.ts

```diff
@@ -210,7 +211,8 @@ export const fetchSuggestions = async (
   profilesEnabled = false
 ) => {
   if (isValidAddress(recipient)) {
-    const ens = await fetchReverseRecord(recipient);
+    const address = getAddress(recipient);
```

Comment:
i believe you already have this method inside `fetchReverseRecord` https://github.com/rainbow-me/rainbow/pull/3583/commits/92956432853a96365b37f28ad8d50b02398552df#diff-5ccad94d6e752008b7c6a23d6dff4f5b42ea30564d8f4aafaf5fa1553fa94a25R944